### PR TITLE
Navbar improvement

### PR DESCRIPTION
**Pull request by iraycd**
There has been few improvement in bottom top in navbar.
This sloves the problem when the page bottom of the above li is clicked.

### CODE REVIEW

**Comment by schlawinski** [comment]
works perfect for me in firefox. IE and chrome doesn´t.
changing your new added line 34 to:
<code> var currentPosition = $(window).scrollTop();</code>
worked better for me.

File: js/js.js [hunk 1]
```diff
@@ -31,9 +31,18 @@ jQuery(document).ready(function ($) {
...
 
     function goToByScroll(dataslide) {
-        htmlbody.animate({
-            scrollTop: $('.slide[data-slide="' + dataslide + '"]').offset().top
-        }, 2000, 'easeInOutQuint');
+        var currentPosition = -1*($('html').offset().top);
+        var clickedPosition = $('.slide[data-slide="' + dataslide + '"]').offset().top;
+        if(clickedPosition>currentPosition){
+            htmlbody.animate({
+                scrollTop: $('.slide[data-slide="' + dataslide + '"]').offset().top
```
Comment (Dan20) [inline js/js.js:38]: In my testing, I needed to add a positive offset here to mirror the negative on line 43.

**Comment by brandonjspencer** [comment]
To correct the issue I added a  + 1 value to line 38:

scrollTop: $('.slide[data-slide="' + dataslide + '"]').offset().top + 1

It just needed that extra little boost!

**Comment by keroberts** [comment]
I have fixed the waypoint inaccuracy by using waypoints v2 and specifying two waypoints. 

The slide needs to PASS the waypoint to trigger. And because of this a single offset waypoint would only ever get triggered in one direction.

So by using two waypoints you can cover both up and down.

I've also added script to dynamically get the .menu height, so the header can be responsive, and not have to rely on using padding on the slide to move the content from under the header.

See my post for all the code:

https://github.com/jalxob/cool-kitten/issues/38

File: js/js.js (hunk 1 above)
Comment (2dvisio) [inline js/js.js:38]: I can confirm the positive offset.

**Pull request closed without merging**